Gems/AtomLyIntegration/AtomViewportDisplayInfo/Code/Source/AtomViewportDisplayInfoSystemComponent.cpp

```diff
@@ -378,7 +392,11 @@ namespace AZ::Render
         );
     }
 
+#if defined (CARBONATED)
+    void AtomViewportDisplayInfoSystemComponent::DrawFramerate(bool drawFpsOnly)
```

Comment:
can we rename drawFPSOnly to drawAverageOnly

Reply:
Updated